### ISSUE TITLE
Is the any RoadMap

### ISSUE THREAD

**Issue by Prathapkarumanchi** [ainsoft-org/coursify.ai#2]
Hi, Is there any road map available? 
And also add nodejs version that support installing packages.

**Comment by uwussimo** [comment]
Yes, We do have a roadmap, currently, this project is commercial though, we will make our decisions to make it open source after the 1st release, but we still maintain the codebase open to public

**Comment by uwussimo** [comment]
We are open for any suggestions))

**Comment by Prathapkarumanchi** [comment]
How about sharing it with contributors?  something like notion template, Trello or discord server

**Comment by uwussimo** [comment]
I will set it up in couple of days

**Comment by uwussimo** [comment]
@Prathapkarumanchi https://github.com/ainsoft-org/coursify.ai/discussions
GitHub discussion is the way to go for now))
